src/components/SavedProjectPlan/StandardCard.jsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Flex,
+  Heading,
+  Image,
+} from "@chakra-ui/react";
+
```

Comment:
This is how nextImage is currently working. You need to have a parent component that has position="relative"